src/app/actions/__tests__/currency.test.ts

```
@@ -0,0 +1,167 @@
+/**
+ * Tests for getCurrencyPrice (live) and getCachedCurrencyPrice (60s TTL).
+ *
+ * The cache shim in @/utils/no-cache is module-scoped, so we jest.resetModules()
+ * between tests and re-require the SUT to start each test with a cold cache.
+ */
+
+import { AccountType } from '@/interfaces'
```

ESLint

[error] '`@/interfaces`' import is restricted from being used. Import from a specific file instead of the '`@/interfaces`' barrel — barrels force the bundler to load every re-export and hurt build perf. See CLAUDE.md.

(no-restricted-imports)